pkg/volume/util/util_windows.go

```diff
@@ -0,0 +1,30 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"path/filepath"
+)
+
+func isAbs(path string) bool {
+	// on Windows, filepath.IsAbs will not return True for paths prefixed with a slash, even
+	// though they can be used as absolute paths (https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats).
+	return filepath.IsAbs(path) || (len(path) > 0 && (path[0] == '\\' || path[0] == '/'))
```

Comment:
why do we want to relax atomic writer to also permit leading `\` or drive letters for atomic writer on Windows? if `/`-prefixed paths work on windows (and according to this doc link, they do?) can we keep atomic writer consistently using `/`-prefixed paths? all the API-specified relative path mount points are required to be specified that way.

Reply:
ditto.